.changeset/brown-crabs-tap.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-backend-module-bitbucket-server': minor
+'@backstage/backend-defaults': minor
```

Comment:
Potentially ship these as `patch`?

Reply:
We spoke about this and agreed that minor is good!

Reply:
perfect, thank you!